Vault-16367: add support RSA padding scheme pkcs1v15 for encryption (#16368)
hashicorp/vault: website/content/api-docs/secret/transit.mdx

- `plaintext` `(string: <required>)` – Specifies **base64 encoded** plaintext to
  be encoded.

> - `padding_scheme` `(string: "oaep")` – When using a RSA key, specifies the RSA
>   padding scheme to use for encryption. Supported signature types are:

Reviewer: ```suggestion
- `padding_scheme` `(string: "oaep")` – Specifies the RSA encryption padding
  scheme for RSA keys. Must be one of the following supported signature types:
```

Style correction: write in active voice

- `ciphertext` `(string: <required>)` – Specifies the ciphertext to decrypt.

> - `padding_scheme` `(string: "oaep")` – When using a RSA key, specifies the RSA
>   padding scheme to use for decryption. Supported signature types are:

Reviewer: ```suggestion
- `padding_scheme` `(string: "oaep")` – Specifies the RSA decryption padding
  scheme for RSA keys. Must be one of the following supported signature types:
```

- `ciphertext` `(string: <required>)` – Specifies the ciphertext to re-encrypt.

> - `padding_scheme` `(string: "oaep")` – When using a RSA key, specifies the RSA
>   padding scheme to use for re-encryption. Supported signature types are:

Reviewer: ```suggestion
- `padding_scheme` `(string: "oaep")` – Specifies the RSA re-encryption padding
  scheme for RSA keys. Must be one of the following supported signature types:
```
